app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js

```diff
@@ -189,7 +189,7 @@ class MigrationsCompletedList extends React.Component {
                         onClick={e => {
                           e.stopPropagation();
 
-                          redirectTo(`/migration/plan/${plan.id}`);
+                          redirectTo(`/plan/${plan.id}`);
```

Comment:
do you not want to use the connected react router's [push method](https://github.com/supasate/connected-react-router/blob/master/FAQ.md#with-storedispatch) now? This should enable the history to be tracked in redux state.

Reply:
Yep i know about this. My mindset about tthis is PR, is to make as little changes as possible (we use the push, goBack etc. methods in ui-classic) to minimize change of breaking something.

But if you will require this changes i will happily do it

Reply:
@Hyperkid123 i think it would be a nice to test this feature if it's not too much additional (otherwise, yes, fine w/ waiting). 

I will try to pull this branch down and do some more testing on my side soon...

Reply:
OK i will do it, its very simple change anyway. The worst thing will be finding all the places where it should be used 😄 .